package.json

```diff
@@ -114,40 +114,40 @@
     "@rollup/plugin-typescript": "^12.1.1",
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.0.1",
-    "@types/node": "^22.8.4",
+    "@types/node": "^22.10.1",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
     "@types/use-sync-external-store": "^0.0.6",
-    "@typescript-eslint/eslint-plugin": "^8.12.2",
-    "@typescript-eslint/parser": "^8.12.2",
-    "@vitest/coverage-v8": "^2.1.4",
-    "@vitest/eslint-plugin": "^1.1.7",
-    "@vitest/ui": "^2.1.4",
+    "@typescript-eslint/eslint-plugin": "^8.17.0",
+    "@typescript-eslint/parser": "^8.17.0",
+    "@vitest/coverage-v8": "^2.1.8",
+    "@vitest/eslint-plugin": "^1.1.14",
+    "@vitest/ui": "^2.1.8",
     "esbuild": "^0.24.0",
     "eslint": "8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-import-resolver-alias": "^1.1.2",
     "eslint-plugin-import": "^2.31.0",
-    "eslint-plugin-jest-dom": "^5.4.0",
+    "eslint-plugin-jest-dom": "^5.5.0",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-react": "^7.37.2",
-    "eslint-plugin-react-compiler": "19.0.0-beta-6fc168f-20241025",
+    "eslint-plugin-react-compiler": "19.0.0-beta-df7b47d-20241124",
     "eslint-plugin-react-hooks": "^5.0.0",
-    "eslint-plugin-testing-library": "^6.4.0",
+    "eslint-plugin-testing-library": "^7.0.0",
     "immer": "^10.1.1",
     "jsdom": "^25.0.1",
     "json": "^11.0.0",
-    "prettier": "^3.3.3",
-    "react": "19.0.0-rc.0",
-    "react-dom": "19.0.0-rc.0",
+    "prettier": "^3.4.2",
+    "react": "19.0.0-rc.1",
+    "react-dom": "19.0.0-rc.1",
     "redux": "^5.0.1",
-    "rollup": "^4.24.3",
+    "rollup": "^4.28.0",
     "rollup-plugin-esbuild": "^6.1.1",
     "shelljs": "^0.8.5",
     "shx": "^0.3.4",
-    "typescript": "^5.6.3",
+    "typescript": "^5.7.2",
     "use-sync-external-store": "^1.2.2",
```

Comment:
React 19 just got released, this needs to be updated as I don't think 1.2.2 supports R19

Reply:
This is just a dev dependency, and the peer dependency is specified as a range, so it shouldn't be the actual problem.

We will update it for the next release anyway.

Reply:
Actually, "devDependecies" is not included in the published package.json.
See: https://unpkg.com/browse/zustand@5.0.2/package.json

Reply:
Ah ok, ignore me.  I doubt this is the issue indeed.  I'll wait for the next release before trying R19 again.

Reply:
> I doubt this is the issue indeed. I'll wait for the next release before trying R19 again.

So, do you have an actual issue? The next release doesn't change it. Please report your issue.

Reply:
I just fixed the issue, one of my tests was throwing `TypeError: Cannot read properties of null (reading 'useSyncExternalStore')`, turned out to be a dependency issue, fixed by adding zustand to devDeps...